core/templates/domain/summary/exploration-summary-backend-api.service.spec.ts

```diff
@@ -15,96 +15,119 @@
 /**
  * @fileoverview Unit tests for ExplorationSummaryBackendApiService.
  */
-import { UpgradedServices } from 'services/UpgradedServices';
-
-require('domain/summary/exploration-summary-backend-api.service.ts');
-require('services/csrf-token.service.ts');
-
-describe('Exploration Summary Backend Api Service', function() {
-  var ExplorationSummaryBackendApiService = null;
-  var $httpBackend = null;
-  var $rootScope = null;
-  var CsrfService = null;
-  var AlertsService = null;
-
-  beforeEach(angular.mock.module('oppia'));
-  beforeEach(angular.mock.module('oppia', function($provide) {
-    var ugs = new UpgradedServices();
-    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
-      $provide.value(key, value);
-    }
-  }));
-  beforeEach(angular.mock.inject(function($injector, $q) {
-    ExplorationSummaryBackendApiService = $injector.get(
-      'ExplorationSummaryBackendApiService');
-    $httpBackend = $injector.get('$httpBackend');
-    CsrfService = $injector.get('CsrfTokenService');
-    $rootScope = $injector.get('$rootScope');
-    AlertsService = $injector.get('AlertsService');
-
-    spyOn(CsrfService, 'getTokenAsync').and.callFake(function() {
-      var deferred = $q.defer();
-      deferred.resolve('sample-csrf-token');
-      return deferred.promise;
+import { HttpClientTestingModule, HttpTestingController } from
+  '@angular/common/http/testing';
+import { TestBed, fakeAsync, flushMicrotasks } from '@angular/core/testing';
+
+import { AlertsService } from 'services/alerts.service';
+import { CsrfTokenService } from 'services/csrf-token.service.ts';
+import { ExplorationSummaryBackendApiService } from
+  'domain/summary/exploration-summary-backend-api.service.ts';
```

Comment:
Can you sort imports as per the comment here?

Reply:
done!